api/v1/controllers/subjects.js

```diff
@@ -33,6 +34,9 @@ const apiErrors = require('../apiErrors');
 const redisSubjectModel = require('../../../cache/models/subject');
 const sampleStoreFeature =
                   require('../../../cache/sampleStore').constants.featureName;
+const jobType = require('../../../jobQueue/setup').jobType;
+const jobWrapper = require('../../../jobQueue/jobWrapper');
+const WORKER_TTL = 30000;
 const ZERO = 0;
```

Comment:
ok so in that case should we have a 30 second timeout setting to be used in any jobs that send back a response, and the current default timeout to be used in asynchronous jobs?
Or can we just do 30 seconds for everything? Do bulkUpsert jobs need longer than that?